packages/office-ui-fabric-react/src/components/DetailsList/DetailsHeader.base.tsx

```diff
@@ -245,24 +245,36 @@ export class DetailsHeaderBase extends BaseComponent<IDetailsHeaderBaseProps, ID
                         id={`${this._id}-check`}
                         aria-label={ariaLabelForSelectionColumn}
                         aria-describedby={
-                          ariaLabelForSelectAllCheckbox && !this.props.onRenderColumnHeaderTooltip ? `${this._id}-checkTooltip` : undefined
+                          !isCheckboxHidden
+                            ? ariaLabelForSelectAllCheckbox && !this.props.onRenderColumnHeaderTooltip
+                              ? `${this._id}-checkTooltip`
+                              : undefined
+                            : ariaLabelForSelectionColumn && !this.props.onRenderColumnHeaderTooltip
+                            ? `${this._id}-checkTooltip`
+                            : undefined
                         }
-                        data-is-focusable={!isCheckboxHidden}
+                        data-is-focusable={!isCheckboxHidden ? true : undefined}

```

Comment:
```suggestion
                        data-is-focusable={!isCheckboxHidden || undefined}
```

Verified via Chrome DevTools:

```shell
var foo = false
15:22:43.400 undefined
15:22:46.100 foo || undefined
15:22:46.102 undefined
15:22:55.181 !foo || undefined
15:22:55.183 true
15:22:57.781 foo = true
15:22:57.783 true
15:23:00.557 !foo || undefined
15:23:00.559 undefined
```

Can you double check my logic tho?

Reply:
Good suggestion, makes sense to me!